.github/workflows/build.yaml

```diff
@@ -0,0 +1,79 @@
+name: Tag, Build and Push Image
+
+on:
+  push:
+    branches:
+    - develop
+    paths-ignore:
+    - 'README.md'
```

Comment:
Similar to https://github.com/DataBiosphere/consent-ontology/pull/519, should we add LICENSE.txt?